app/trust_registry_facade.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+from typing import Literal, Union, List
+from fastapi.exceptions import HTTPException
+
+import requests
+
+TRUST_REGISTRY_URL = os.getenv("TRUST_REGISTRY_URL", "http://localhost:8001/")
+
+logger = logging.getLogger(__name__)
+
+
+async def actor_has_role(
+    actor_id: str, role: str = Union[Literal["issuer"], Literal["verifier"]]
```

Comment:
I think you can just do:
```suggestion
    actor_id: str, role: str = Literal["issuer", "verifier"]
```

Reply:
Also maybe we can make one role type and import that everywhere to make it easier to add roles over time?

```
Role = Literal["issuer", "verifier"]
```